vpr/src/device/rr_spatial_lookup.h

```diff
@@ -155,14 +167,20 @@ class RRSpatialLookup {
      *                          vtr::Point<int>(1, 2),
      *                          SOURCE,
      *                          TOP);
+     * ```
      *
-     * Note: currently this function only accepts SOURCE/SINK nodes. May unlock for the other types 
+     * @note currently this function only accepts SOURCE/SINK nodes. May unlock for the other types 
      * depending on needs
      *
      * TODO: Consider to make a high-level API to duplicate the nodes for large blocks. 
```

Comment:
Maybe the TODO comment should be outside the doxygen comments so it is a note to developers rather than API users?